langgraph_bot/nodes/tnode/generate_image_node.py

```
@@ -0,0 +1,39 @@
+            headers={"Authorization": f"Bearer {secret_key}"},
+            params={"model": "gptimage",
+                    "quality": "high",
+                    "width": 1600,
+                    "height": 900
+            }
+        )
+        if response.status_code != 200:
+            # print(f"Error occured during Image Generation {response.status_code} - {response.content}")
+            raise Exception(f"Error occured during Image Generation {response.status_code} - {response.content}")
+        return {"generated_image": response.content}
+    except Exception as e:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)